src/marqo/vespa/vespa_client.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, current_generation: int, wanted_generation: int, converged: b
     def __init__(self, config_url: str, document_url: str, query_url: str,
                  content_cluster_name: str, default_search_timeout_ms: int = 1000, pool_size: int = 10,
                  feed_pool_size: int = 10, get_pool_size: int = 10, delete_pool_size: int = 10,
-                 partial_update_pool_size: int = 10):
+                 partial_update_pool_size: int = 10, max_search_offset: int = 10000, max_search_limit: int = 1000):
```

Comment:
Where did we get 10000 and 1000 as the limits?

Reply:
I got them from here: https://github.com/marqo-ai/marqo/blob/mainline/src/marqo/api/configs.py

Do you think we should refer to them or just make these two required?